Cabal/src/Distribution/Simple/Register.hs

```diff
@@ -174,8 +174,10 @@ registerAll pkg lbi regFlags ipis =
       for_ ipis $ \installedPkgInfo ->
         -- Only print the public library's IPI
         when
-          ( packageId installedPkgInfo == packageId pkg
-              && IPI.sourceLibName installedPkgInfo == LMainLibName
+          ( packageId installedPkgInfo
+              == packageId pkg
+              && IPI.sourceLibName installedPkgInfo
+                == LMainLibName
```

Comment:
This is a pure reformatting, right? That's unexpected.

Reply:
Yeh, that looks a bit weird to me. (But so do many things `fourmolu` comes up with.)

Reply:
Ok, looks like it's resolved now...